keymaps/terminal.json

```diff
@@ -1,10 +1,26 @@
 {
-  "atom-workspace": {
+  ".platform-darwin atom-workspace": {
     "cmd-shift-t": "terminal:open"
   },
-  "terminal-view": {
+  ".platform-darwin terminal-view": {
     "cmd-c": "terminal:copy",
     "cmd-v": "terminal:paste",
     "cmd-k": "terminal:clear"
+  },
+  ".platform-win32 atom-workspace": {
+    "alt-shift-t": "terminal:open"
+  },
+  ".platform-win32 terminal-view": {
+    "ctrl-c": "terminal:copy",
```

Comment:
I don't think we want to bind to Ctrl-C on Windows, as that's a standard shortcut for interrupting a program when running in a terminal.

Reply:
On linux too. I wanted to stay close of mac keybinding until we find better ideas, will replace by #62 proposition `ctrl` `shift` `c`/`v`/`k`
`cmd` `shift` `k` on mac could solve what you fix with 713fe60d74b3b96b20c40aab04e57352d03a886e